Assets/RGLUnityPlugin/Scripts/RadarObjectClassProvider.cs

```diff
@@ -0,0 +1,109 @@
+// Copyright 2024 Robotec.ai.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using UnityEngine;
+using System;
+using System.Collections.Generic;
+
+namespace RGLUnityPlugin
+{
+    [Serializable]
+    public struct RadarObjectClass
+    {
+        [field: SerializeField]
+        [field: Tooltip("Radar object class for given entity ids range")]
+        public RGLRadarObjectClass ClassId  { get; private set; }
+
+        [field: SerializeField]
+        [field: Tooltip("The beginning of entity id range")]
+        public int EntityIdRangeBegin  { get; private set; }
```

Comment:
Can entity id have negative value? If not, you can consider an attribute here and for `EntityIdRangeEnd` below.

Reply:
The entity id field is an integer so it could have a negative value.